lib/components/MapView.js

```diff
@@ -648,7 +648,29 @@ class MapView extends React.Component {
         this.props.onRegionChange(nativeEvent.region, details);
       }
     } else if (this.props.onRegionChangeComplete) {
-      this.props.onRegionChangeComplete(nativeEvent.region, details);
+      if (Platform.OS === 'ios') {
+        const { latitude, longitude } = nativeEvent.region;
+        return new Promise((resolve, reject) => {
+          this._runCommand('getAddressFromCoordinates', [
```

Comment:
I don't think we want to call `getAddressFromCoordinates` every single time the region changes. Can we expose this as a method on Mapview?

Reply:
Alright will make as method props.

Reply:
One other thing that might be worth considering is returning all the results instead of pulling the first one. Although I'm not sure how much data is in each result so it might be overkill if the first returned result is the most relevant.